qadence/backends/pyqtorch/config.py

```diff
@@ -12,14 +12,14 @@
     flatten,
     scale_primitive_blocks_only,
 )
-from qadence.types import AlgoHEvo, Interaction
+from qadence.types import AlgoHEvo
 
 logger = get_logger(__name__)
 
 
 def default_passes(config: Configuration) -> list[Callable]:
     return [
-        lambda circ: add_interaction(circ, interaction=config.interaction),
+        add_background_hamiltonian,
```

Comment:
Does it mean that now, all digital gates get background interaction ?

Reply:
No, but it does mean that all circuits go through the `add_background_hamiltonian` and consequently all leaf blocks in that circuit go through the `_analog_to_hevo`, but there you can see that only the three selected `Analog` ones are changed into an `HamEvo` with the background interaction, and all other blocks are returned unchanged.

Reply:
Ah right, this is adding missing background interaction to the analog blocks for simulation in PyQTorch. OK I get it. Might be worth adding a comment.